--- barrage-1.0.7/Makefile.am.orig	2023-04-29 17:33:51.000000000 +0200
+++ barrage-1.0.7/Makefile.am	2024-06-15 19:39:25.754308614 +0200
@@ -8,7 +8,7 @@ rpm: dist
 
 desktopdir = $(datadir)/applications
 desktop_DATA = barrage.desktop
-icondir = $(datadir)/icons
+icondir = $(datadir)/pixmaps
 icon_DATA = barrage48.png
 
 EXTRA_DIST = BUGS README COPYING AUTHORS ChangeLog INSTALL NEWS TODO \
@@ -18,7 +18,7 @@ EXTRA_DIST = BUGS README COPYING AUTHORS
 # the right way to handle it? At least it works... as long as $datadir
 # does not contain plus signs. :-)
 barrage.desktop:
-	@sed -e "s+Icon=+Icon=$(datadir)/icons/barrage48.png+" \
+	@sed -e "s+Icon=+Icon=$(icondir)/barrage48.png+" \
 	< barrage.desktop.in > barrage.desktop
 
 ACLOCAL_AMFLAGS = -I m4
